package.json

```
@@ -32,11 +32,10 @@
     "commit": "cz",
     "dev": "vite",
     "format": "prettier --write . --ignore-path .gitignore",
```

```suggestion
    "dev": "vite",
    "lint": "eslint . --fix --ext js,cjs,mjs --ignore-path .gitignore",
    "lint:check": "eslint . --ext js,cjs,mjs --ignore-path .gitignore",
```

I would say, remove format, lint:format, lint:fix as well and change lint to what lint:fix is doing

---

We are using `lint:check` in the github action (I guess that will change once the husky integration is in place).

---

`lint:check` is still unchanged

---

Sorry, my brain is malfunctioning today. I thought you asked about `lint:check` 😅 It sounds good to have `lint` do the fixing, and `lint:check` do the checking 👍